pages/common/httpflow.md

```diff
@@ -0,0 +1,24 @@
+# httpflow
+
+> A command line utility to capture and dump HTTP streams.
+> More information: https://github.com/six-ddc/httpflow.
+
+- Capture traffic on all interfaces:
+
+`httpflow -i any`
+
+- Use a bpf-style capture to filter the results:
+
+`httpflow host host httpbin.org or host baidu.com`
```

Comment:
This is the `expression`? I'm not sure about this, but the argument should be formatted like this:
```suggestion
`httpflow {{host host httpbin.org or host baidu.com}}`
```

Reply:
Duplicated `host` there at the beginning of the token syntax?